src/cdk/table/table.ts

```diff
@@ -544,32 +544,6 @@ export class CdkTable<T> implements AfterContentChecked, CollectionViewer, OnDes
     this.updateStickyColumnStyles();
   }
 
-  /**
-   * Sets the header row definition to be used. Overrides the header row definition gathered by
-   * using `ContentChild`, if one exists. Sets a flag that will re-render the header row after the
-   * table's content is checked.
-   * @docs-private
-   * @deprecated Use `addHeaderRowDef` and `removeHeaderRowDef` instead
-   * @breaking-change 8.0.0
-   */
-  setHeaderRowDef(headerRowDef: CdkHeaderRowDef) {
-    this._customHeaderRowDefs = new Set([headerRowDef]);
-    this._headerRowDefChanged = true;
-  }
-
-  /**
-   * Sets the footer row definition to be used. Overrides the footer row definition gathered by
-   * using `ContentChild`, if one exists. Sets a flag that will re-render the footer row after the
-   * table's content is checked.
-   * @docs-private
-   * @deprecated Use `addFooterRowDef` and `removeFooterRowDef` instead
-   * @breaking-change 8.0.0
-   */
-  setFooterRowDef(footerRowDef: CdkFooterRowDef) {
-    this._customFooterRowDefs = new Set([footerRowDef]);
-    this._footerRowDefChanged = true;
-  }
```

Comment:
These methods seem fairly easy to migrate. Should we add them to the `property-names` ng-update schematics?

<sup>I realize `property-names` might be ambiguous, but in reality it handles property access expressions such as `myTablesetHeaderRowDef`